docs/content/experience/works.json

```
@@ -0,0 +1,20 @@
+[
+  {
+    "place": "University of Malaga",
+    "positions": [
+      {
+        "position": "Researcher",
+        "description": "Small description"
```

```suggestion
        "description": "Small description",
        "skills": [] # A list of skills
```

TODO: Complete descriptions in this `json`